src/modules/Header/components/Header.js

```diff
@@ -69,8 +70,18 @@ const HeaderBarLower = styled.div`
 `
 
 const HeaderUpperRight = styled.div`
-  padding: 14px 20px;
-  line-height: 18px;
+  display: flex;
+  flex-wrap: wrap;
+  padding-right: 20px;
+`
+
+const LanguageBar = styled.div`
+  padding: 14px 0;
```

Comment:
Pixels here too. It would be better to use predifined classes provided by Tailwind CSS.